recipes/picaso/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "picaso" %}
+{% set version = "3.2.2" %}
+{% set python_min = "3.7" %}
```

Comment:
Conda-forge doesn't support Python 3.7. You can remove this and let the automatic min Python be used instead.

```suggestion
```

Reply:
Oh thanks! Didn't know that. I just removed python min for both `virga-exo` and `picaso` in their meta.yaml files.